src/z3c/form/form.rst

```diff
@@ -922,12 +922,12 @@ When the form renders, the label has now changed:
   >>> addForm = PersonAddForm(root, TestRequest())
   >>> addTemplate(addForm)
   >>> addForm.update()
-  >>> print(testing.render(addForm, './/xmlns:div[2][@class="row"]'))
+  >>> print(testing.render(addForm, './/xmlns:div[2][@class="row"]'))  # doctest: +NOPARSE_MARKUP
```

Comment:
Could you comment on what `NOPARSE_MARKUP` is meant to be doing? I'm not familiar with that flag, and I don't see it listed as a flag in [the documentation](https://docs.python.org/3/library/doctest.html#option-flags), nor have I been able to find it any any other zopefoundation repos.

Reply:
`NOPARSE_MARKUP` is a custom flag of `z3c.form`, it is registered here:  https://github.com/zopefoundation/z3c.form/blob/440eb18c541b88f64e88be9b66fbc27748f75af2/src/z3c/form/outputchecker.py#L44

It is used to select the parser for the markup:

https://github.com/zopefoundation/z3c.form/blob/440eb18c541b88f64e88be9b66fbc27748f75af2/src/z3c/form/outputchecker.py#L98-L108

As the default parser now returns the HTML tail which follows the `xpath` selection not parsing the markup helps the tests to succeed.